src/groups/z_bmq/z_bmqa/z_bmqa_session.t.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright 2014-2023 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// bmqa_session.t.cpp
+#include <bmqa_session.h>
+#include <z_bmqa_session.h>
+
+// MWC
+#include <mwcu_memoutstream.h>
+
+// BDE
+#include <bsl_ios.h>
+#include <bsl_string_view.h>
+
+// TEST DRIVER
+#include <mwctst_testhelper.h>
+
+#include <demo.h>
+
+// CONVENIENCE
+using namespace BloombergLP;
+using namespace bsl;
+
+// ============================================================================
+//                                    TESTS
+// ----------------------------------------------------------------------------
+
+static void test1_session()
```

Comment:
We're going to need more basic tests for each one of the APIs that we create here.